database/seed.js

```diff
@@ -0,0 +1,38 @@
+const faker = require('faker');
+const db = require('../database/model.js');
+const dataFormat = require('../helpers/dataFormat.js');
+
+
+const populate = function() {
+
+  // db.dropDatabase();
+
+  const dummyData = [];
+
+  const neighborhoods = ['Last Hearth', 'Crofter\'s Village', 'Hornwood', 'Wolfswood', 'Cerwyn'];
+
+  const cities = ['Ramsgate', 'Dreadfort', 'Winterfell', 'White Harbor', 'Barrowton'];
+
+    for (let i = 0; i < 100; i++) {
+
+      const fakeHome = {
+        property_id: '',
+        streetAddress: faker.address.streetAddress(),
+        streetName: faker.address.streetName(),
+        state: 'The North',
+        zipcode: faker.address.zipCode(),
+        neighborhood: neighborhoods[Math.random() * (neighborhoods.length - 1)],
```

Comment:
Might want to add Math.floor to randomly choosing neighborhood?

Reply:
Great catch! I went back and fixed it